Tests/ttLib/tables/data/_c_m_a_p_format_14.ttx

```diff
@@ -3,9 +3,9 @@
 
   <cmap>
     <tableVersion version="0"/>
-    <cmap_format_14 platformID="0" platEncID="5" format="14" length="49" numVarSelectorRecords="2">
+    <cmap_format_14 platformID="0" platEncID="5">
       <map uvs="0xfe00" uv="0x30" name="zero.slash"/>
-      <map uvs="0xfe01" uv="0x30" name="None"/>
+      <map uvs="0xfe01" uv="0x30" name=""/>
```

Comment:
how about, instead of an empty string, we don't output the `name` attribute in the TTX dump when it is `None`, and similarly assume that name is `None` when the attribute is not present?

Reply:
Ah, that's not a bad suggestion. Let me see.

Reply:
also maybe, shouldn't the unicode variation selector "uvs" attribute be written _after_ the base unicode character value "uv" in the sequence of xml attributes for the `map` element?
That's how one writes them, e.g. 0x30 0xFE00, not viceversa

Reply:
Yeah, also a fair point.